app/experimenter/static/js/design-page.js

```diff
@@ -0,0 +1,494 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import {
+  Button,
+  Container,
+  Row,
+  Col,
+  FormControl,
+  FormLabel
+} from "react-bootstrap";
+
+const branchesDiv = document.getElementById("react-branches-form");
+
+function PrefForm(props) {
```

Comment:
I also forgot to mention all these form components should go in their own JS files.